app/controllers/graphql_controller.rb

```diff
@@ -0,0 +1,50 @@
+class GraphqlController < ApplicationController
+  # If accessing from outside this domain, nullify the session
```

Comment:
This file was added by default with the graphql gem.

Reply:
thank you for clarifying